cmd/api/src/database/agi.go

```diff
@@ -86,47 +86,74 @@ func (s *BloodhoundDB) DeleteAssetGroup(ctx context.Context, assetGroup model.As
 	})
 }
 
+func (s *BloodhoundDB) getAssetGroupMemberCount(ctx context.Context, assetGroupID int32) (int, error) {
+    var (
+        latestCollection model.AssetGroupCollection
+        result           = s.preload(model.AssetGroupCollectionAssociations()).
+                             WithContext(ctx).
+                             Where("asset_group_id = ?", assetGroupID).
+                             Order("created_at DESC").
```

Comment:
I noticed we have a bespoke db call here over the `GetLatestAssetGroupCollection`, is the original implementation not valid? If so we might want to consider moving this to that method and calling it here so the other callers get fixed as well. If both are valid, I think we might want to shy away from the bespoke db hit.

Reply:
The former might be the case as `GetLatestAssetGroupCollection` has been returning empty asset groups despite new custom members being added. I will adapt my fix into that function.